addons/sale_project/static/src/components/project_right_side_panel/components/project_profitability_section.js

```diff
@@ -0,0 +1,92 @@
+import { useService } from "@web/core/utils/hooks";
+import { Component, useState } from "@odoo/owl";
+import { formatFloat, formatFloatTime } from "@web/views/fields/formatters";
+
+export class ProjectProfitabilitySection extends Component {
+
+    static props = {
+        revenue: Object,
+        labels: Object,
+        formatMonetary: Function,
+        onProjectActionClick: Function,
+        onClick: Function,
+        projectId: Number,
+        context: Object,
+    };
+    static template = "sale_project.ProjectProfitabilitySection";
+
+
+    setup() {
+        this.orm = useService("orm");
+        this.actionService = useService("action");
+        this.state = useState({
+            isFolded: true,
+            displayLoadMore: null,
+        });
+        this.sale_items = [];
+        this.foldableSection = this._getFoldableSection();
+    }
+
+    _getFoldableSection() {
+        return ['materials', 'service_revenues'];
+    }
```

Comment:
Could this not be sent by the backend, rather than being hard-coded here?

Reply:
I suppose we can, but since those are immutable value, i did not see the point of doing an orm call for that. Do you think it's worth it ?